### PR TITLE
added additional details related to admonitions

### PR DESCRIPTION
**Pull request by fusion94**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [X] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

**Comment by fusion94** [comment]
update

File: docs/gemstones/markdown-demo-v2.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ tags:
 
 # Overview
 
+## Background
+* Use [Markdown](https://daringfireball.net/projects/markdown).
```
Comment (alemorvan) [inline docs/gemstones/markdown-demo-v2.md:15]:
```suggestion

* Use [Markdown](https://daringfireball.net/projects/markdown).
```

File: docs/gemstones/markdown-demo-v2.md [hunk 2]
```diff
@@ -53,10 +58,33 @@ At other times, you need bulleted or numbered lists:
...
 
+### Admonitions
+
+Admonitions, also known as call-outs, are an excellent choice for including side content without significantly interrupting the document flow. Material for MkDocs provides several different types of admonitions and allows for the inclusion and nesting of arbitrary content.
+
 !!! TIP
 
     Pencils and staplers are really old-school.
 
+#### Usage
```
Comment (alemorvan) [inline docs/gemstones/markdown-demo-v2.md:69]:
```suggestion
#### Usage

```

File: docs/gemstones/markdown-demo-v2.md [hunk 3]
```diff
@@ -53,10 +58,33 @@ At other times, you need bulleted or numbered lists:
...
+Admonitions follow a simple syntax: a block starts with !!!, followed by a single keyword used as a type qualifier. The content of the block follows on the next line, indented by four spaces:
+
+!!! note
+
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod
+    nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor
+    massa, nec semper lorem quam in massa.
+
+#### Changing the title
```
Comment (alemorvan) [inline docs/gemstones/markdown-demo-v2.md:78]:
```suggestion
#### Changing the title

```

File: docs/gemstones/markdown-demo-v2.md [hunk 4]
```diff
@@ -53,10 +58,33 @@ At other times, you need bulleted or numbered lists:
...
+### Admonitions
+
+Admonitions, also known as call-outs, are an excellent choice for including side content without significantly interrupting the document flow. Material for MkDocs provides several different types of admonitions and allows for the inclusion and nesting of arbitrary content.
+
 !!! TIP
 
     Pencils and staplers are really old-school.
 
+#### Usage
+Admonitions follow a simple syntax: a block starts with !!!, followed by a single keyword used as a type qualifier. The content of the block follows on the next line, indented by four spaces:
```
Comment (alemorvan) [inline docs/gemstones/markdown-demo-v2.md:70]:
```suggestion
Admonitions follow a simple syntax: a block starts with `!!!`, followed by a single keyword used as a type qualifier. The content of the block follows on the next line, indented by four spaces:
```

**Comment by fusion94** [comment]
applied

> Thx @fusion94 I just added some formating tips that you can commit.
> 
> Thx.

**Review by alemorvan — APPROVED** [review]

**Pull request merged**